index.js

```diff
@@ -0,0 +1,8 @@
+const burger = document.getElementById('fa-bars');
+const hideMenu = document.getElementById('closeIcon');
+const navLinks = document.getElementById('desktop-nav');
+
+const hideMenuFunc = (arg) => { navLinks.style.right = arg; };
+burger.onclick = () => hideMenuFunc('0');
+hideMenu.onclick = () => hideMenuFunc('-100%');
+navLinks.onclick = () => hideMenuFunc('-100%');
```

Comment:
- [ ] Well done implementing the functionality for hiding the menu button👏👏😎, However, this project is missing the functionality for displaying the menu when the user clicks the hamburger button in mobile view, kindly consider implementing the JS functionality for that.